nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -53,44 +55,59 @@ NavfnPlanner::NavfnPlanner()
   declare_parameter("tolerance", rclcpp::ParameterValue(0.0));
   declare_parameter("use_astar", rclcpp::ParameterValue(false));
 
-  tf_ = std::make_shared<tf2_ros::Buffer>(get_clock());
-  auto timer_interface = std::make_shared<tf2_ros::CreateTimerROS>(
-    rclcpp_node_->get_node_base_interface(),
-    rclcpp_node_->get_node_timers_interface());
-  tf_->setCreateTimerInterface(timer_interface);
-  tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*tf_);
+  // Setup the global costmap
+  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>(
+    "global_costmap", nav2_util::add_namespaces(std::string{get_namespace()},
+    "global_costmap"));
+  costmap_ = nullptr;
```

Comment:
not sure this is needed, but if you want to keep it, maybe add to the initialization list

Reply:
What are you saying is not needed?

Reply:
Ah sorry, the preview shows lines 59-62. I think you mean the `costmap_ = nullptr`, which yeah I can add to the constructors init list :+1: